name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/dashboard/SharpeRatioConfig.java

```diff
@@ -0,0 +1,85 @@
+package name.abuchen.portfolio.ui.views.dashboard;
+
+import org.eclipse.jface.action.IMenuManager;
+import org.eclipse.jface.dialogs.InputDialog;
+import org.eclipse.jface.window.Window;
+import org.eclipse.swt.widgets.Display;
+
+import name.abuchen.portfolio.model.Dashboard;
+import name.abuchen.portfolio.ui.Messages;
+import name.abuchen.portfolio.ui.PortfolioPlugin;
+import name.abuchen.portfolio.ui.util.SimpleAction;
+
+/**
+ * Configuration of the Sharpe Ratio widget
+ */
+public class SharpeRatioConfig implements WidgetConfig
+{
+    private WidgetDelegate<?> delegate;
+    private int riskFreeIRR;
+
+    public SharpeRatioConfig(WidgetDelegate<?> delegate)
+    {
+        this.delegate = delegate;
+        String riskFreeIRRStr = this.delegate.getWidget().getConfiguration().get(Dashboard.Config.RISKFREE_IRR.name());
+        if (riskFreeIRRStr != null)
+        {
+            try
+            {
+                this.riskFreeIRR = Integer.parseInt(riskFreeIRRStr);
+            }
+            catch (NumberFormatException ex)
+            {
+                this.riskFreeIRR = 0;
+                PortfolioPlugin.info(ex.getLocalizedMessage());
+            }
+        }
```

Comment:
If riskFreeIRRStr is null, the parseInt method will throw a NumberFormatException, then we jump from try to catch.

```suggestion
        {
        try
        {
            this.riskFreeIRR = Integer.parseInt(riskFreeIRRStr);
        }
        catch (NumberFormatException ex)
        {
            this.riskFreeIRR = 0;
            PortfolioPlugin.info(ex.getLocalizedMessage());
        }
```